main.go

```diff
@@ -93,9 +83,40 @@ func main() {
 	log.Infof("\nChangelog:")
 	log.Printf(changelog)
 
+	envmanConfigs, err := envman.GetConfigs()
+	if err != nil {
+		failf("Failed to load envman configs, error: %s", err)
+	}
+
+	if envmanConfigs.EnvBytesLimitInKB > 0 {
+		envBytesLimit := envmanConfigs.EnvBytesLimitInKB * 1024
+
+		if len(changelog) > envBytesLimit {
+			log.Warnf("Changelog content exceeds the maximum allowed size to set in an environment variable. (%dKB)", envmanConfigs.EnvBytesLimitInKB)
+			log.Warnf("The changelog's content will be trimmed to fit the maximum allowed size.")
+			log.Warnf("It is possible to modify the limit by following this guide: https://devcenter.bitrise.io/tips-and-tricks/increasing-the-size-limit-of-env-vars")
```

Comment:
It would be helpful to reccomend using the changelog file to the user as an alternative.